### ISSUE TITLE
Skydock can't start anymore since Docker 1.11

### ISSUE THREAD

**Issue by xflash** [crosbymichael/skydock#101]
Hi,

Since an update to docker 1.11.0 (API 1.23) I can't start anymore skydock :

> [info] 1461666680 skydock: loading plugins from /plugins/default.js
> [info] 1461666680 skydock: skydns URL: http://172.17.0.27:8080
> [debug] 1461666680 skydock: starting restore of containers
> [fatal] 1461666680 skydock: error restoring containers: json: cannot unmarshal string into Go value of type docker.State
> json: cannot unmarshal string into Go value of type docker.State

Apparently due to some change in the Docker status event  ?

Thanks for your help !

**Comment by SergeyDubovitsky** [comment]
I have the same problem:
`:~$docker logs skydock 
[info] 1461680379 skydock: loading plugins from /plugins/default.js
[info] 1461680379 skydock: skydns URL: http://172.17.0.2:8080
[debug] 1461680379 skydock: starting restore of containers
[fatal] 1461680379 skydock: error restoring containers: json: cannot unmarshal string into Go value of type docker.State
json: cannot unmarshal string into Go value of type docker.State
`

`:~$ docker --version
Docker version 1.11.0, build 4dc5990`

**Comment by emilb** [comment]
Yes, there has been an API change since docker 1.11.x
https://docs.docker.com/engine/reference/api/docker_remote_api/#v1-23-api-changes

I think it's the GET /containers/json that causes the problem now. @crosbymichael, would you like to have a look? I've come to like and use skydock and skydns for a lot of my projects.

**Comment by vany-egorov** [comment]
You can use native docker 
```{r, engine='bash'}
docker network create --driver bridge my_network_01;
docker network ls;

docker run --name my-container-01-01  -d \
		--net=my_network_01 \
		-h my-container-01-01 \
		my-image-01;

docker run --name my-container-02  -d \
		--net=my_network_01 \
		-h my-container-02-02  \
		my-image-01;

docker network inspect my_network_01;

# or you can use connect instead of --net
docker network connect network_01 my-container-01-01;
docker network connect network_01 my-container-01-01;
docker network inspect my_network_01;

# and second network
docker network create --driver bridge my_network_02;
docker network ls;

docker run --name my-container-02-01  -d \
		-h my-container-02-01 \
		my-image-02;

docker network connect network_01 my-container-02-01;
docker network connect network_02 my-container-01-01;
docker network connect network_02 my-container-02-01;
```

**Comment by athlordJojo** [comment]
So the project is dead, if this bug is not fixed, right ?

**Comment by crosbymichael** [comment]
@athlordJojo not dead but there is no need fore this project anymore for people who are running the new version of docker that does this all internally.  I believe 1.9+ will have automatic DNS discovery for your containers and swarm adds discover + and overlay network.  

I'm just keeping the project around for existing users that have not upgraded yet

**Comment by crosbymichael** [comment]
You honestly should not be using this anymore if you are running docker 1.11+.  Docker is handling this all for you now and it is much simpler.

**Comment by athlordJojo** [comment]
yeah we are going to update our software too. Thanks for the information.
